lib/templates/reconciliationText.js

```diff
@@ -155,6 +155,9 @@ class ReconciliationText {
       // Skip externally managed items if externally_managed is false
       if (this.EXTERNALLY_MANAGED_ITEMS.includes(attribute) &&
         templateConfig.externally_managed === false) {
+        consola.warn(
+          `The following attributes were skipped because they can only be updated when the template is externally managed: ${this.EXTERNALLY_MANAGED_ITEMS.join(", ")}.`
+        );
```

Comment:
This is good, but if we include it here (in the iteration itself) it will be printed each time there is a skipped item.
Now it's only once because there is only one item `downloadable_as_docx`. But if we ever add more items the message will be repeated as many times as items are skipped

Reply:
I have adjusted the logic so that the warning is only shown outside of the reduce.